crossterm_cursor/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![deny(unused_imports)]
```

Comment:
Strange wy it complains about unused import like it is quite obvious that they are re-exported for external use

Reply:
Where it complains? Maybe you just swapped `allow` and `deny` for a moment. `allow` says that unused imports are okay. `deny` says that it's an error and Travis will fail. Did add it so now new unused imports are introduced (CI check).